StPetersburg/InEtn66217.xml

```diff
@@ -0,0 +1,231 @@
+                    <title type="complete" ref="LIT2285Saqoqa" xml:lang="gez">ሰቆቃወ፡ ድንግል፡</title>
+                    <textLang mainLang="gez"></textLang>
+                    <note>Text is distributed according to the days of the week. The name of <persName ref="PRS12884HaylaG" role="owner"></persName> is
+                  mentioned on <locus target="#49r"></locus>.</note>
+                  </msItem>
+
+                  <msItem xml:id="ms_i4">
+                    <locus from="49r" to="63r"/>
+                     <title type="complete" ref="LIT1099Anapho" xml:lang="gez">አኰቴተ፡ ቁርባን፡ ዘእግዝእትነ፡ ማርያም፡ ዘደረሰ፡ አባ፡ ሕርያቆስ፡ በሀገረ፡ ብሕንሳ፡</title>
+                     <textLang mainLang="gez"/>
+                     <note>The name of <persName ref="PRS12885WaldaS" role="owner"></persName> is mentioned on <locus target="#63r"></locus>.</note>
+                  </msItem>
+
+               </msContents>
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <material key="parchment"/>
+                        </support>
+
+                        <extent>
+                           <measure unit="leaf">70</measure>
+                           <measure unit="leaf" type="blank">12</measure>
+                           <locus target="#1 #8 #9 #10 #32 #33 #65 #66 #67 #68 #69 #70"></locus>
+                           <dimensions type="outer" unit="mm">
+                              <height>175</height>
+                              <width>130</width>
+                           </dimensions>
+                        </extent>
+                        </supportDesc>
+                        <layoutDesc>
+                          <layout columns="2" writtenLines="19"></layout>
+                        </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script"/>
+                        <seg type="ink">Black, red</seg>
+                        <desc/>
+                     </handNote>
+                  </handDesc>
+                  <decoDesc>
+                    <decoNote xml:id="d1" type="ornamentation">
+                      <locus target="#11r"></locus>
+                      <desc>Ornamental band with white, red and black interwoven lines.</desc>
+                    </decoNote>
+                    <decoNote xml:id="d2" type="ornamentation">
+                      <locus target="#49r"></locus>
+                      <desc>Ornamental band with black and white interwoven lines.</desc>
+                    </decoNote>
+                  </decoDesc>
+                  <additions>
+                    <list>
+                      <item xml:id="a1">
+                        <locus target="#7r"></locus>
+                        <desc type="GuestText">A table of shadows length during daytime in different months.</desc>
```

Comment:
shadow length?

Reply:
in the catalogue description it is in plural

Reply:
"the length of shadows" maybe? It really sounds strange to me.

Reply:
done

Reply:
I think you need to add, in whatever way, the definitions "calendar" and "computus" (also to the key words), or make it to "CalendaricNote".